src/helpers.ts

```diff
@@ -66,22 +66,27 @@ function printDoc(gridResult: [], columns: ColumnModel[], gridName: string): voi
     documentToPrint.document.close();
 }
 
-function exportFile(gridResult: [], columns: ColumnModel[]): void {
+function exportFile(gridResult: [], columns: ColumnModel[], gridName: string): void {
     const csvFile = getCsv(gridResult, columns);
 
     const blob = new Blob(['\uFEFF' + csvFile], {
         type: 'text/csv;charset=utf-8;',
     });
 
+    let fileName = 'data'
```

Comment:
You can build the full file name from here, instead of 'data', you can initialize this value as 'data.csv'.